boot/src/main/java/io/edpn/backend/application/controller/BootTradeModuleController.java

```diff
@@ -1,15 +1,22 @@
 package io.edpn.backend.application.controller;
 
-import io.edpn.backend.trade.application.controller.v1.DefaultTradeModuleController;
-import io.edpn.backend.trade.domain.service.v1.FindCommodityMarketInfoService;
-import io.edpn.backend.trade.domain.service.v1.FindCommodityService;
-import io.edpn.backend.trade.domain.service.v1.LocateCommodityService;
+import io.edpn.backend.trade.adapter.web.TradeModuleController;
+import io.edpn.backend.trade.application.port.incomming.commoditymarketinfo.GetFullCommodityMarketInfoUseCase;
+import io.edpn.backend.trade.application.port.incomming.locatecommodity.LocateCommodityUseCase;
+import io.edpn.backend.trade.application.port.incomming.validatedcommodity.FindAllValidatedCommodityUseCase;
+import io.edpn.backend.trade.application.port.incomming.validatedcommodity.FindValidatedCommodityByFilterUseCase;
+import io.edpn.backend.trade.application.port.incomming.validatedcommodity.FindValidatedCommodityByNameUseCase;
 import org.springframework.web.bind.annotation.RestController;
 
 
 @RestController
-public class BootTradeModuleController extends DefaultTradeModuleController {
-    public BootTradeModuleController(FindCommodityMarketInfoService findCommodityMarketInfoService, LocateCommodityService locateCommodityService, FindCommodityService findCommodityService) {
-        super(findCommodityMarketInfoService, locateCommodityService, findCommodityService);
+public class BootTradeModuleController extends TradeModuleController {
```

Comment:
I'm not sure why these Boot*Controller classes exist - the `@RestController` annotation could just be on `TradeModuleController` with correct classpath scanning (or `@Bean` annotated public methods in the `ControllerConfig` class).

At the moment, ControllerConfig exists, but is empty.

This isn't a huge issue, and certainly doesn't block a merge, but something for us (as a group) to consider.

Reply:
I agree with @officialyinsane here, I don't see a _need_ for this class.